integration/test/cwl_util.go

```diff
@@ -104,6 +105,65 @@ func DeleteLogGroupAndStream(logGroupName, logStreamName string) {
 	}
 }
 
+// ValidateLogsInOrder takes a log group, log stream, a list of specific log lines and a timestamp.
+// It should query the given log stream for log events, and then confirm that the log lines that are
+// returned match the expected log lines. This also sanitizes the log lines from both the output and
+// the expected lines input to ensure that they don't diverge in JSON representation (" vs ')
+func ValidateLogsInOrder(t *testing.T, logGroup, logStream string, logLines []string, since time.Time) {
+	log.Printf("Checking %s/%s since %s for %d expected logs", logGroup, logStream, since.UTC().Format(time.RFC3339), len(logLines))
+	cwlClient, clientContext, err := getCloudWatchLogsClient()
```

Comment:
Doing that means changing the function to accept a `*testing.T` parameter, like 
```go
func getCloudwatchLogsClient(t *testing.T) {}
```
I am indifferent on how we do it. The end result is the same. I just didn't see much reason to pass the testing struct down another level.

Reply:
I am fine in passing down a struct at a deeper level since it would reduce code and main reason was: Do we let each function handles custom respond to the error? Since it is not that's why I would prefer passing down; however, its fine for you to keep it since the end result is the same.

Reply:
I disagree with the current approach for the CloudWatch metrics client creation in `agent_util.GetCWClient()` because it uses `log.Fatal` which makes it harder to differentiate that from the individual test executions.